services/x-net/setup_executor_compose.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+cat > /subspace/docker-compose.yml << EOF
+version: "3.7"
+
+services:
+  node:
+    image: vedhavyas/subspace-node:latest
```

Comment:
https://hub.docker.com/layers/subspace-node/vedhavyas/subspace-node/latest/images/sha256-e3f01474fb07c38dc8d378b68afbbe3c05ad20a2684ffa172c7589d4549fff2e?context=explore how's it different from GitHub version ?

Reply:
this is a testnet that do not have docker images. So I created one for x-net. Also we use ghcr.io instead of docker hub

Reply:
Doesn't `jun-18` image contain all the changes necessary?

Reply:
I think @liuchengxu has more changes coming in. We need to update the images once those docker images are up anyway which may not be part of jun-18. I'll let @liuchengxu weigh in on this

Reply:
Yeah, `jun-18` was enough before, but not now due to this critical executor bugfix https://github.com/subspace/subspace/pull/627 added very recently. I have also built a docker image on ghcr.io in my fork and updated the executor node in the cloud. Additionally, I also have to make another image with [a substrate patch](https://github.com/subspace/subspace/commit/754eca4bfed70931443027fb420f80090283f66b) to help debug https://github.com/paritytech/substrate/issues/11732. Currently, I just ensure the images are fine and replace them directly on the server, happy to have some instructions on the upgrade/patching in a more organized way and follow them later.